python_code_review/object_orienting_programming.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import pytz
+
+
+class Account:
+
+	@staticmethod
+	#creates a static method, meaning u don't need the self parameter anymore and this static method can be called using Account._current_time
+	def _current_time():
+		utc_time = datetime.datetime.utcnow()
+		return pytz.utc.localize(utc_time)
+
+	def __init__(self, name, balance):
+		self.name = name
+		self.transaction_list = []
+		self.balance = balance
+		print("Account created for {} with balance of {}".format(self.name, self.balance))
```

Comment:
Use single quotes.

Since you're using python3, use f'' strings. The print statement instead should be

```
print(f'Account created for {self.name} with balance of {self.balance}')
```

Reply:
There are other examples like this below. Correct them as well

Reply:
got it.